index.bs

```diff
@@ -10454,86 +10462,62 @@ Constructors</h5>
-	::
-		Implements the audio processing algorithm for the
-		{{AudioWorkletProcessor}}.
-
-		The {{process()}} method is called
-		synchronously by the audio <a>rendering thread</a> at
-		every <a>render quantum</a>, if {{AudioWorkletNode}} is
-		[=actively processing=].
-
-		<div class="note">
-			The return value of this method controls the lifetime
-			of the {{AudioWorkletProcessor}}'s associated
-			{{AudioWorkletNode}}.
-
-			This lifetime policy can support a variety of approaches
-			found in built-in nodes, including the following:
-
-			* Nodes that transform their inputs, and are active only
-				while connected inputs and/or script references exist. Such
-				nodes SHOULD return <code>false</code> from
-				{{process()}} which allows the presence or absence of
-				connected inputs to determine whether the {{AudioWorkletNode}} is
-				[=actively processing=].
-
-			* Nodes that transform their inputs, but which remain active
-				for a <a>tail-time</a> after their inputs are disconnected. In
-				this case, {{process()}} SHOULD return
-				`true` for some period of time after
-				<code>inputs</code> is found to contain zero channels. The
-				current time may be obtained from the global scope's
-				{{AudioWorkletGlobalScope/currentTime}} to
-				measure the start and end of this tail-time interval, or the
-				interval could be calculated dynamically depending on the
-				processor's internal state.
-
-			* Nodes that act as sources of output, typically with a
-				lifetime. Such nodes SHOULD return `true` from
-				{{process()}} until the point at which they are no
-				longer producing an output.
-
-			Note that the preceding definition implies that when no
-			return value is provided from an implementation of
-			{{process()}}, the effect is identical to returning
-			<code>false</code> (since the effective return value is the falsy
-			value {{undefined}}). This is a reasonable behavior for
-			any {{AudioWorkletProcessor}} that is active only when it has
-			active inputs.
-		</div>
-
-		<pre class=argumentdef for="AudioWorkletProcessor/process(inputs, outputs, parameters))">
-			inputs:
- 				The input audio buffer from the incoming connections provided by the user agent. It has type <code>sequence&lt;sequence&lt;Float32Array>></code>.<code class="nobreak">inputs[n][m]</code> is a {{Float32Array}} of audio samples for the \(m\)th channel of the \(n\)th input. While the number of inputs is fixed at construction, the number of channels can be changed dynamically based on [=computedNumberOfChannels=].
-
-				If there are no [=actively processing=] {{AudioNode}}s connected to the \(n\)th input of the {{AudioWorkletNode}} for the current render quantum, then the content of <code>inputs[n]</code> is an empty array, indicating that zero channels of input are available. This is the only circumstance under which the number of elements of <code>inputs[n]</code> can be zero.
+The [=process()=] callback function is called
```

Comment:
Instead of explaining how this is invoked here, referring https://webaudio.github.io/web-audio-api/#rendering-loop directly might be clearer and more specific.

Reply:
This was pretty much in the original, but I can reference the rendering loop.

Reply:
Yeah, I think referencing the algorithm itself would be more thorough.

Reply:
I will remove this and just reference the algorithm.